WD-Firmware/psp/api_inc/psp_asm_macros.h

```diff
@@ -14,8 +14,16 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 */
+#ifndef __PSP_ASM_MACROS_H__
+#define __PSP_ASM_MACROS_H__
 
-.if __riscv_xlen == 64
+#include "psp_specific_rtos.h"
```

Comment:
1, dont use include, check why its here.
2. dont use #define __PSP_ASM_MACROS_H__ in asm. we need to use ".if" and if can not use then dont use it.

Reply:
1. done. moved to the appropriate place (i.e. to the S files that include psp_asm_macros.h)
2. done - removed